sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/VisualStudioTestResultsFileParser.java

```diff
@@ -36,6 +36,7 @@
 
   @Override
   public void accept(File file, UnitTestResults unitTestResults) {
+    LOG.debug("The current user dir is '{}'.", System.getProperty("user.dir"));
```

Comment:
Nothing to log here? `NCover3ReportParser.java` reported it's progress on the fly inside the document. At least log handleCountersTag?